otx/algorithms/segmentation/configs/ocr_lite_hrnet_x_mod3/template.yaml

```diff
@@ -29,14 +29,14 @@ hyper_parameters:
         default_value: 8
```

Comment:
Please revert base config change in current hrnet.

Reply:
I can, but 300 default epochs looks like overhead. Maybe we can decrease this number?